build/gulp/ts.js

```diff
@@ -157,6 +158,35 @@ gulp.task('ts-check-public-modules', gulp.series('ts-copy-modules', function() {
         .pipe(compileTS({ allowSyntheticDefaultImports: true }));
 }));
 
+gulp.task('test-ts', gulp.series(
+  (callback) => {
+    exec(`cd testing/typescript && npm i --no-audit --no-fund`, (e, out, err) => {
+      console.log(out, err);
```

Comment:
It's better to use `spawn` instead of `exec`. It allows controlling stdout so we'll get the real-time output instead of the `console.log` on the done callback. See the example of usage [here](https://github.com/DevExpress/devextreme-cli/blob/0aef326559c4168120ccacbdaddfd0e2f15e2c77/packages/devextreme-cli/src/utility/run-command.js).

Reply:
done